src/index.ts

```diff
@@ -179,17 +181,8 @@ class Envoy {
     this.ws_server.on("connection", async (socket, request) => {
       // path should contain the HHA ID and Agent ID so we can do some checks and alert the
       // client-side if something is not right.
-      log.silly("Incoming connection from %s", request.url);
       const url = new URL(request.url, "http://localhost");
 
-      socket.on("message", (data) => {
```

Comment:
why are we removing this?

Reply:
I thought I'd added that for my own logging, so I took it out. I'll put it back.